CHANGELOG.md

```diff
@@ -8,7 +8,9 @@
 
 ##### Bug Fixes
 
-* None.  
+* Verify container portal when checking dependency target membership.  
+  [izaakschroeder](https://github.com/izaakschroeder)
+  [#513](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

Comment:
`https://github.com/CocoaPods/CocoaPods/issues/513`

Reply:
Oh. Ack. Whoops sorry I'll migrate it to this repo thanks 👍